server/routes/googleCalendarAPI.js

```diff
@@ -6,16 +6,55 @@ const router = new express.Router();
 
 // get user's availability from google calendar
 router.post("/api/calendar/availability", async (req, res) => {
-  const day = req.query.day;
-  console.log(day);
+  const day = new Date(req.query.day);
   if (req.body.members.length > 1) {
     // team availability
   } else {
     try {
+      const availability = [];
       const usr = await db.User.findById(req.body.members[0]);
```

Comment:
this is a small thing, but I much prefer having words printed out clearly for easier reading. Eg. `usr` vs `user`